e2e/playwright/actions/src/tests/library/library-actions.spec.ts

```diff
@@ -0,0 +1,444 @@
+/*!
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { expect } from '@playwright/test';
+import {
+  Utils,
+  ApiClientFactory,
+  test,
+  SitesApi,
+  FavoritesPageApi,
+  timeouts,
+  DataTableComponent,
+  SnackBarComponent,
+  AdfConfirmDialogComponent,
+  AcaHeader,
+  MatMenuComponent,
+  TrashcanApi
+} from '@alfresco/playwright-shared';
+import { Site } from '@alfresco/js-api';
+
+test.describe('Library actions ', () => {
+  const apiClientFactory = new ApiClientFactory();
+  const notMemberString = 'Not a member';
+  const managerRole = 'Manager';
+  const contributorRole = 'Contributor';
+  const consumerRole = 'Consumer';
+  const joinButton = 'Join';
+  const libraryJoinedMessage = 'Library joined';
+  const requestToJoinMessage = 'Request sent to join this library';
+  const cancelJoinRequestButton = 'Cancel Join Request';
+  const cancelJoinRequestMessage = 'Canceled the request to join the library';
+  const leaveLibraryButton = 'Leave Library';
+  const leaveDialogTitle = 'Leave this library?';
+  const leaveDialogContent = 'Leaving will remove your access.';
```

Comment:
same here like above

Reply:
done in [aa865dc](https://github.com/Alfresco/alfresco-content-app/pull/3545/commits/aa865dcf243348da373c4560117ab99260fb3fe8)